ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -32,6 +32,21 @@ message Event {
   reserved "exercised";
 }
 
+// Result of evaluating an interface view consisting of the view definition
+// and the resulting value.
+message InterfaceViewResult {
+  Identifier interface_id = 1;
+  Record value = 2;
+}
+
+// The 0-based index of the filters that matched the request. This is useful for
+// demultiplexing streams on the client side. This is only set for
+// the transaction stream & the ACS stream but not for the
+// transaction tree stream.
+message FilterIndices {
+  repeated uint32 indices = 1;
+}
+
```

Comment:
@cocreature : is this a left-over piece? I don't see it used anywhere.

Reply:
Yes it’s leftover. Unfortunately, I think I dropped the usage too eagerly thinking that with the simplified APIs we no longer need it. The problematic case is when we have subscriptions for multiple ifaces with `include_create_blob = true`. In that case, we lose the information on which interface matched.

I’d say we have to handle this case somehow. How about rather than going via indices we include `repeated Identifier implemented_interfaces` which is the intersection of matched filters & interfaces implemented by the template?

Reply:
Sounds good. I'd though go for `matched_interfaces` then.